cli/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -131,6 +133,7 @@ impl PathEnvironment {
     ///
     /// This method should only be used to create [CoreEnvironment]s for a [PathEnvironment].
     /// To modify the environment, use the [PathEnvironment] methods instead.
+    #[allow(dead_code)]
```

Comment:
suggestion: ignore dead_code warning, don't silence it

```suggestion
```
this will be used when #620 is merged it sneaked into another PR by accident.

clippy might not e happy with it here but we'd be removing it (or likely forget to) anyway

Reply:
Fixed